tests/bluetooth/tester/src/mesh.c

```diff
@@ -454,7 +459,8 @@ static void config_prov(uint8_t *data, uint16_t len)
 
 static void provision_node(uint8_t *data, uint16_t len)
 {
-	const struct mesh_provision_node_cmd *cmd = (void *) data;
+	const struct mesh_provision_node_cmd *cmd = (void *)data;
```

Comment:
These changes regarding provisioner should be merged with previous commit that adds support for provisioner.